eng/Version.Details.xml

```diff
@@ -37,7 +37,7 @@
       Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
       All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.2" CoherentParentDependency="Microsoft.AspNetCore.Testing">
```

Comment:
Add `CoherentParentDependency="Microsoft.NET.Sdk.Razor"` to line 47 as well

Reply:
It's needed because there is no darc subscription from dotnet/runtime to this repo - without it, that dependency will never get updated

Reply:
Also note that because the razor sdk is so old, this downgrades a bunch of things to 6.0.0-alpha.1.21070.2

Reply:
Probably need to update that entry too

Reply:
Thanks, updated to 7.0 dependencies, marking as resolved.